common/autoware_overlay_rviz_plugin/autoware_overlay_rviz_plugin/src/signal_display.cpp

```diff
@@ -47,7 +47,7 @@ SignalDisplay::SignalDisplay()
   property_top_ = new rviz_common::properties::IntProperty(
     "Top", 10, "Top position of the overlay", this, SLOT(updateOverlayPosition()));
   property_signal_color_ = new rviz_common::properties::ColorProperty(
-    "Signal Color", QColor(94, 130, 255), "Color of the signal arrows", this,
+    "Signal Color", QColor("25, 18, 89"), "Color of the signal arrows", this,
```

Comment:
```suggestion
    "Signal Color", QColor(QString("#00E678")), "Color of the signal arrows", this,
```
![Screenshot from 2024-02-06 11-37-18](https://github.com/autowarefoundation/autoware.universe/assets/10751153/3ef0ea28-f5ad-460a-980d-486cc0e9de94)
 
Current color is darker than the proposed `#3F2EE3` from the [Autoware website color palette](https://coolors.co/3f2ee3-00c8ff-00e678-ff5c0f-00002a-0094ff). But even when I updated it to that color, it still lacked the contrast.

https://coolors.co/contrast-checker/3f2ee3-000000

This color doesn't work on black backgrounds :(

Instead let's use `#00E678` from [the palette](https://coolors.co/3f2ee3-00c8ff-00e678-ff5c0f-00002a-0094ff) then.

https://coolors.co/contrast-checker/00E678-000000

It has much better contrast.

Also you can use hex strings directly in QColor() constructor.